plasmapy/particles/atomic.py

```diff
@@ -6,7 +6,6 @@
     "common_isotopes",
     "electric_charge",
     "half_life",
-    "integer_charge",
```

Comment:
I'm wondering if it'd be worth it to have a `charge_number` function here too.  Maybe a follow-up PR...

Reply:
Oh wait, we already have one.  Nevermind.